src/locales/en-US/translation.json

```
@@ -188,6 +188,19 @@
         }
       }
     },
+    "snowflake": {
+      "name": "snowflake",
+      "description": "Display the syntaxes for a specific Discord snowflake",
```

I would prefer to word this as
```suggestion
      "description": "Display the syntax for the timestamp of a Snowflake (most IDs within Discord)",
```